CHANGELOG.md

```diff
@@ -101,6 +101,10 @@
   [#9962](https://github.com/Kong/kong/pull/9962)
 - **OpenTelemetry**: Support scoping with services, routes and consumers.
   [#10096](https://github.com/Kong/kong/pull/10096)
+- **Statsd**: Add `tag` configuration
```

Comment:
I think the name `tag` is misleading as it implies and actual value. What this option supports is a tagging _style_. [(as described in the promehteus docs](https://github.com/Kong/kong-ee/pull/4207))

I'd suggest to re-name this to `tag_style`

Reply:
good idea

Reply:
I've been thinking about it for a while， using `tag` means tagging off or using some tag style, but `tag` style means using which one tag style. So I prefer using `tag` as this option.

Reply:
I also vote for `tag_style`(or more precise, `metrics_tag_style`?) , because semantically a `tag_style = null` would be enough to tell the user that there is no tag style configured in this plugin. And what's more, there is already a `tags` field in many core entities, so to avoid misleading & conflicts I think rename this field is a better way